### PR TITLE
SDK maxAttempts now configurable through env, plugin logs now debug

### PR DESCRIPTION
**Pull request by dkershner6**
https://discord.com/channels/983865673656705025/1168586969543082237

### CODE REVIEW

File: packages/open-next/src/adapters/server-adapter.ts [hunk 1]
```diff
@@ -24,20 +23,40 @@ declare global {
 
 const CACHE_BUCKET_REGION = process.env.CACHE_BUCKET_REGION;
 
+function parseS3ClientConfigFromEnv(): S3ClientConfig {
+  return {
+    region: CACHE_BUCKET_REGION,
+    logger: awsLogger,
+    maxAttempts: parseNumberFromEnv(process.env.AWS_SDK_S3_MAX_ATTEMPTS),
```
Comment (khuezy) [inline packages/open-next/src/adapters/server-adapter.ts:30]: Is the "defaults to 3" an AWS default config?

Comment (dkershner6) [reply packages/open-next/src/adapters/server-adapter.ts:30]: Yessir.  Effectively `undefined` === 3.  I could change the docs to be "defaults to whatever the SDK defaults to" if that is preferred, but I think that is the direction I would take.

**Review by khuezy — APPROVED** [review]

File: packages/open-next/src/adapters/server-adapter.ts (hunk 1 above)
Comment (khuezy) [reply packages/open-next/src/adapters/server-adapter.ts:30]: Thx, that's fine, I just wanted to make sure we didn't miss the defaults.

**Comment by conico974** [comment]
Frank was faster than you, he already merged this one https://github.com/sst/open-next/pull/311

**Pull request merged**